recipe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "imbalanced-ensemble" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - path: requirements.txt
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/imbalanced-ensemble-{{ version }}.tar.gz
+    sha256: 77b53ade6749b76464919d2314589ee32c1cfb7b782afac1642c9980c7cf62f4
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [py>39]
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - numpy >=1.16.0
+    - scipy >=1.9.1
+    - pandas >=1.1.3
+    - joblib >=0.11
+    - scikit-learn >=1.3.0
+    - matplotlib-base >=3.3.2
+    - seaborn >=0.13.2
```

Comment:
shouldn't this be `seaborn >=0.11.2`

Reply:
Yes, thank you! Must of been left over when I was playing with dependencies.